src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -313,5 +313,16 @@ public void inserimentoCodiceDelegaACaricoDellImpresaAPI(String codiceDelega){
 
     }
 
+    public boolean siVisualizzaUnaDelegaPG() {
+        try {
+            By menuDelega = By.xpath("//tr[contains(@class,'MuiTableRow-root css-g76qb5')]");
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(menuDelega));
```

Comment:
By default we usually put this **10 seconds** because it shouldn't take so much time to load a button if so there is a problem behind the scene